testing/jormungandr-testing-utils/Cargo.toml

```
@@ -63,6 +63,8 @@ tracing-subscriber = "0.2"
 tracing = "0.1"
 netstat2 = "0.9"
 multiaddr = { package = "parity-multiaddr", version = "0.11" }
+keynesis = "1"
+bincode = "1"
```

It would be good to be consistent with the version spec elsewhere:

```suggestion
bincode = "1.3"
```